security-fuzzing-handbook/build.sh

```diff
@@ -0,0 +1,7 @@
+
+docker run --rm \
+             --volume "$(pwd):/data" \
+	     --user $(id -u):$(id -g) \
+	     pandoc/extra fuzzing-handbook.md \
+	     -o handbook-fuzzing.pdf \
```

Comment:
Either
```suggestion
	     pandoc/extra handbook-fuzzing.md \
	     -o handbook-fuzzing.pdf \
```
or
```suggestion
	     pandoc/extra fuzzing-handbook.md \
	     -o fuzzing-handbook.pdf \
```

Reply:
It's intentional as is, and I'd prefer to keep it as is -- it makes it easier to use the tab on the command line.